crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -252,6 +252,17 @@ where
 }
 
 impl<'w, T: Resource> Res<'w, T> {
```

Comment:
Just throwing an idea out there: maybe we should call this something else to avoid the ambiguity entirely?
(ex: `duplicate()`)

Reply:
Yep, that's my preference here too.

Reply:
Hmm actually within the context of requiring `Res::clone` to resolve the ambiguity, I think thats nice and clear, given that we're cloning the Res. I think we should merge this as-is.